packages/webapp/pages/onboarding.tsx

```diff
@@ -254,6 +278,15 @@ export function OnboardPage(): ReactElement {
       return setActiveScreen(OnboardingStep.Extension);
     }
 
+    if (
+      !options?.clickExtension &&
+      installDesktopExperiment &&
+      canUserInstallDesktop &&
+      activeScreen === OnboardingStep.Extension
+    ) {
+      return setActiveScreen(OnboardingStep.InstallDesktop);
+    }
+
```

Comment:
`canUserInstallDesktop` is very important because due to the odd way to archive this feature we could not have the `event.prompt` object available.